src/Widget.tsx

```diff
@@ -32,85 +32,80 @@ const Wrapper = styled.div`
     font-size: 16px;
     padding: 36px 50px;
   }
-`
+`;
 declare global {
   interface Window {
-    analytics: any
+    analytics: any;
   }
 }
 
 const SearchWidget = () => {
-  const [showPersonalInfo, setShowPersonalInfo] = React.useState(false)
-  const methods = useForm({ mode: 'onSubmit', reValidateMode: 'onChange' })
-  const [isSubmitting, setSubmitting] = React.useState(false)
+  const [showPersonalInfo, setShowPersonalInfo] = React.useState(false);
+  const methods = useForm({ mode: "onSubmit", reValidateMode: "onChange" });
+  const [isSubmitting, setSubmitting] = React.useState(false);
 
-  const { handleSubmit, watch } = methods
-  watch()
-  const onSubmit = async ({
+  const { handleSubmit } = methods;
+
+  const clean = (obj: Record<string, string>): Record<string, string> =>
+    Object.fromEntries(Object.entries(obj).filter(([_, v]) => v != null));
+
+  const onSubmit = ({
     hereFor,
     firstName,
     lastName,
     email,
-    phone
+    phone,
   }: {
-    hereFor: string
-    firstName: string
-    lastName: string
-    email: string
-    phone: string
+    hereFor: string;
+    firstName: string;
+    lastName: string;
+    email: string;
+    phone: string;
   }) => {
     if (isSubmitting) return;
-    setSubmitting(true)
-    const requestOptions = {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({
-        here_for: hereFor,
-        first_name: firstName,
-        last_name: lastName,
-        email,
-        phone
-      })
-    }
+    setSubmitting(true);
 
-    const response = await fetch('https://api.solace.health/v1/api/prospects', requestOptions)
-    const data: { id: string } = await response.json()
-    
-    if (data.id) {
-      const redirect = `https://find.solace.health/?p_id=${data.id}`
-      if (window.analytics) {
-        window.analytics.track('FUNNEL_ENTRY', {
-          context: 'MarketingHome',
-          location,
-          redirect_url: redirect
-        })
-      }
-      setTimeout(() => { 
-        console.log(redirect)
-        window.location.assign(redirect);
-      }, 500);
+    const data = clean({
+      here_for: hereFor,
+      first_name: firstName,
+      last_name: lastName,
+      email,
+      phone,
+    });
 
-      setSubmitting(false)
-      return false;
-    }
-  }
+    const formQuery = new URLSearchParams(data);
+    const redirect = `https://find.solace.health/?${formQuery}`;
```

Comment:
Can we remove this slash?

```suggestion
    const redirect = `https://find.solace.health?${formQuery}`;
```